index.html

```diff
@@ -1,17 +1,12 @@
 ---
 ---
+
 <!DOCTYPE html>
 <html lang="en">
-{% include header.html %}
-
-<body>
-    {% include navbar.html %}
-
-    {% include profile.html %}
-    
-    {% include description.html %}
-    
-    {% include projects.html %}
-</body>
+  {% include header.html %}
 
-</html>
\ No newline at end of file
+  <body>
+    {% include navbar.html %} {% include profile.html %} {% include
```

Comment:
I'd suggest leaving these the way they were. It looks cleaner to only have one per line. It makes it easy to figure out the structure of the page that way.